tests/FakeItEasy.Specs/ArgumentMatchingSpecs.cs

```diff
@@ -85,7 +85,7 @@ public static void WhenArgumentsMatchWith2ArgsSuccess(IFoo fake, int result)
                 .x(() => fake = A.Fake<IFoo>());
 
             "And a fake method with 2 parameters is configured to return a value when the arguments match a predicate"
-                .x(() => A.CallTo(() => fake.Bar(0, null))
+                .x(() => A.CallTo(() => fake.Bar(0, string.Empty))

```

Comment:
I suppose technically we could use a `A.Dummy<string>()`.

Reply:
Inside a call configuration? Not sure what that would do... `string.Empty` is fine, I think.

Reply:
Whoops. I put the comment on the wrong line. Although I'm pretty sure it would just return a `""` here.